src/containerapp/azext_containerapp/_validators.py

```diff
@@ -215,3 +220,49 @@ def validate_timeout_in_seconds(cmd, namespace):
     if timeout_in_seconds is not None:
         if timeout_in_seconds < 0 or timeout_in_seconds > 60:
             raise ValidationError("timeout in seconds must be in range [0, 60].")
+
+
+def validate_debug(cmd, namespace):
+    print("Validating...")
+    revision_already_set = bool(namespace.revision)
+    replica_already_set = bool(namespace.replica)
+    container_already_set = bool(namespace.container)
+    _set_debug_defaults(cmd, namespace)
+    if revision_already_set:
+        _validate_revision_exists(cmd, namespace)
+    if replica_already_set:
+        _validate_replica_exists(cmd, namespace)
+    if container_already_set:
+        _validate_container_exists(cmd, namespace)

```

Comment:
These 3 validations looks like client side validation, however, they must call a server side GET to retrieve the actual information to compare.
Given that the performance is not good because too many server side round-trips are made before we can actually attach to a console, could we just remove these 3 checks, and rely on the server side check for the `/debug` API endpoint. So if any of the app, revision, replica, container does not exist, the API returns 404 with an error message, and the client just terminate with that error message.

Reply:
current cli's websocket version won't read response body. let's retain client side validation.